README.md

```
@@ -73,18 +79,29 @@ Parameters for the script are controlled by the following environment variables:
 - `NVIDIAGPU_GPU_FALLBACK_CATALOGSOURCE_INDEX_IMAGE`: custom certified-operators catalogsource index image for GPU package - _required when deploying fallback custom GPU catalogsource_
 - `NVIDIAGPU_NFD_FALLBACK_CATALOGSOURCE_INDEX_IMAGE`:  custom redhat-operators catalogsource index image for NFD package - _required when deploying fallback custom NFD catalogsource_
-- `TEST_LABELS`: ginkgo query passed to the label-filter option for including/excluding tests - _optional_
-- `VERBOSE_SCRIPT`: prints verbose script information when executing the script - _optional_
-- `TEST_VERBOSE`: executes ginkgo with verbose test output - _optional_
-- `TEST_TRACE`: includes full stack trace from ginkgo tests when a failure occurs - _optional_
+
+NVIDIA Network Operator-specific (NNO) parameters for the script are controlled by the following environment variables:
+- `NVIDIANETWORK_CATALOGSOURCE`: custom catalogsource to be used.  If not specified, the default "certified-operators" catalog is used - _optional_
+- `NVIDIANETWORK_SUBSCRIPTION_CHANNEL`: specific subscription channel to be used.  If not specified, the latest channel is used - _optional_
+- `NVIDIANETWORK_BUNDLE_IMAGE`: Network Operator bundle image to deploy with operator-sdk if NVIDIANETWORK_DEPLOY_FROM_BUNDLE variable is set to true.  Default value for bundle image if not set: TBD - _optional when deploying from bundlle_
+- `NVIDIANETWORK_DEPLOY_FROM_BUNDLE`: boolean flag to deploy GPU operator from bundle image with operator-sdk - Default value is false - _required when deploying from bundle_
```

Markdownlint (0.37.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)